src/main/java/org/wildfly/swarm/plugin/utils/ChecksumUtil.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2018 Red Hat, Inc, and individual contributors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.swarm.plugin.utils;
+
+import javax.xml.bind.DatatypeConverter;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+/**
+ * @author Michal Szynkiewicz, michal.l.szynkiewicz@gmail.com
+ * <br>
+ * Date: 8/7/18
+ */
+public class ChecksumUtil {
+
+    private ChecksumUtil() {
+    }
+
+
+    public static String getChecksum(File file) throws IOException, NoSuchAlgorithmException {
+        return calculateChecksum(file);
+    }
+
+    protected static String calculateChecksum(File file) throws NoSuchAlgorithmException, IOException {
+        try (FileInputStream in = new FileInputStream(file)) {
+            byte[] buf = new byte[1024];
+            int len = 0;
+
+            MessageDigest md = MessageDigest.getInstance("SHA1");
+
+            while ((len = in.read(buf)) >= 0) {
+                md.update(buf, 0, len);
+            }
+
+            return toHex(md.digest()).toLowerCase();
+        }
+    }
+
+    private static String toHex(byte[] digest) {
+        return DatatypeConverter.printHexBinary(digest);
```

Comment:
`javax.xml.bind` will disappear from the JDK soon enough :-)

Reply:
replaced with my own code